libbeat/template/processor.go

```diff
@@ -31,7 +31,6 @@ import (
 const DefaultField = false
 
 var (
-	minVersionAlias                   = common.MustNewVersion("6.4.0")
 	minVersionFieldMeta               = common.MustNewVersion("7.6.0")
```

Comment:
Likely something we can also get rid of if Beats 8.0 only support ES 8.0. Same for the ones below.